### PR TITLE
Add `.first ` to bind

### PR DESCRIPTION
**Pull request by reset**
`bind` and `svc` should feel fairly similar. This will add the `first`
field to the bind struct to enable consumers of a service group through
a bind to have access to that field as well

Related to #2253

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/sup/src/templating/context.rs [hunk 1]
```diff
@@ -37,12 +37,16 @@ impl<'a> Binds<'a> {
 
 #[derive(Clone, Debug, Serialize)]
 pub struct BindGroup<'a> {
-    members: Vec<SvcMember<'a>>,
+    pub first: Option<SvcMember<'a>>,
```
Comment (reset) [inline components/sup/src/templating/context.rs]: Optional binds may never populate with members, so we need to have this be an `Option`

**Comment by reset** [comment]
tenor-100976617

If anyone objects feel free to comment after this merge!

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/230177061).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/230177061).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**